engine/hid/src/hid.h

```diff
@@ -35,6 +35,8 @@ namespace dmHID
 {
     /// Constant that defines invalid context handles
     const HContext INVALID_CONTEXT = 0;
+    const uint8_t MAX_GAMEPAD_NAME_LENGTH = 128;
+    const uint8_t MAX_GAMEPAD_NAME_COUNT  = 2;
```

Comment:
We used a max of 128 bytes for the name elsewhere, so I figured we could make these const instead..